swap/cheque.go

```diff
@@ -133,22 +133,22 @@ func (cheque *Cheque) verifyChequeProperties(p *Peer, expectedBeneficiary common
 
 // verifyChequeAgainstLast verifies that the amount is higher than in the previous cheque and the increase is as expected
 // returns the actual amount received in this cheque
-func (cheque *Cheque) verifyChequeAgainstLast(lastCheque *Cheque, expectedAmount *uint256.Uint256) (*uint256.Uint256, error) {
+func (cheque *Cheque) verifyChequeAgainstLast(lastCheque *Cheque, expectedAmount *uint256.Uint256) error {
```

Comment:
It is related. The reason why a possibility in difference between `expectedAmount ` and `actualAmount` was introduced here was to allow small differences in observed honey price (this was never fully implemented though). Anyway a future PR will probably get rid of the `expectedAmount` (and `cheque.Honey`) and instead reintroduce the `actualAmount` as the only value but let's do that after all the big number stuff has been merged to avoid too much breakage.

Reply:
comment removed